manual/03-lists/Transforming.md

```diff
@@ -2,3 +2,46 @@ List to list transformations
 ============================
 
 TODO: `map`, `pluck`, `filter`, `aperture`, `uniq`, `zip`, `partition`
+
+Many times we loop over lists for...
+--
+
+**Transforming**
+```js
+const libs = ["Ramda", "Lodash", "Pramda", "Underscore"]
+const html = []
+for(var i = 0; i < libs.length; i++){
+  html[i] = "<li>"+libs[i]+"</li>"
+}
+console.log(html)
+
+```
+Above code is too verbose and we are prone to making mistakes. Ramda provides a `map` function to achieve
```

Comment:
To my mind, the problem with a `for` loop is it mixes together the iteration and the transformation. The `map` version abstracts away the iteration, so all you have to do is  write the function that transforms each element. 

This leaves aside the fact that `map` is for more than just lists, of course ...